src/polysolve/LinearSolverSympiler.hpp

```diff
@@ -0,0 +1,59 @@
+#ifdef POLYSOLVE_WITH_SYMPILER
+
+#include <polysolve/LinearSolver.hpp>
+#include <sympiler/parsy/cholesky_solver.h>
```

Comment:
Let's move this header to the .cpp. This will require moving the definition of the constructor/destructor to the .cpp of this file as well.

Reply:
Is this possible given the member variables of type `sym_lib::parsy::CSC` and `sym_lib::parsy::SolverSettings`?

Reply:
I took a crack at it -- let me know how it looks